app/utils/extract_content.py

```
@@ -0,0 +1,228 @@
+            all_tables.append(
+                {
+                    "index": i,
+                    "page": int(table.page) if str(table.page).isdigit() else table.page,
+                    "text": table_text,
+                    "rows": df_as_rows,
+                }
+            )
+
+            print(f"  Extracted table {i+1} from page {table.page}")
+
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)